view/frontend/web/template/payment/venmo.html

```diff
@@ -0,0 +1,41 @@
+<!-- Venmo -->
+<!-- ko if: isAllowed() && isBrowserSupported() -->
+<div class="payment-method" data-bind="css: {'_active': (getCode() == isChecked())}" id="payment-method-braintree-venmo">
+    <div class="payment-method-title field choice">
+        <input type="radio"
+               name="payment[method]"
+               class="radio"
+               data-bind="attr: {'id': getCode()}, value: getCode(), checked: isChecked, click: selectPaymentMethod, visible: isRadioButtonVisible()" />
+        <label class="label" data-bind="attr: {'for': getCode()}">
+            <img data-bind="attr: {src: getPaymentMarkSrc(), alt: $t('Venmo')}"
+                 width="60"
+                 class="payment-icon"/>
+            <span text="getTitle()"></span>
+        </label>
+    </div>
+    <div class="payment-method-content">
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() --><!-- /ko -->
+        <!--/ko-->
+
+        <div class="payment-method-billing-address">
+            <!-- ko foreach: $parent.getRegion(getBillingAddressFormName()) -->
+            <!-- ko template: getTemplate() --><!-- /ko -->
+            <!--/ko-->
+        </div>
+
+        <div class="checkout-agreements-block">
+            <!-- ko foreach: $parent.getRegion('before-place-order') -->
+            <!-- ko template: getTemplate() --><!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="actions-toolbar">
+            <div class="primary">
+                <button class="action primary checkout" data-bind="click: clickVenmoBtn" type="submit">
+                    <span>Pay with Venmo</span>
```

Comment:
can we translate?

Reply:
Sure can! (How do you translate in a KO template?)

Reply:
it's awful, but:
`<span data-bind="i18n: 'Pay with Venmo'"></span>`